pkg/v1/sdk/features/plugin/activate.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package plugin
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// FeatureActivateCmd is for activating Features
+var FeatureActivateCmd = &cobra.Command{
```

Comment:
This seems to go against the existing pattern of having these cobra commands in `cmd/cli/plugin`. Any reason why this is different?

Reply:
there seem to be multiple patterns in the framework repo, I was following the builder pattern where the root command exists `cmd/cli/plugin-admin/builder` and the subcommands in `pkg/v1/builder`, but ya in general it makes sense to keep the commands under `cmd/cli/plugin`, will make the changes